src/libraries/System.Private.Xml.Linq/src/System/Xml/Linq/XDocumentType.cs

```diff
@@ -53,14 +53,10 @@ internal XDocumentType(XmlReader r)
         /// <summary>
         /// Gets or sets the internal subset for this Document Type Definition (DTD).
         /// </summary>
-        public string InternalSubset
+        public string? InternalSubset
```

Comment:
This a breaking change and needs to be added to the relevant docs for that.

Reply:
I'll add a label to this PR, I plan to handle the breaking changes separately

Reply:
oh, you already did 😄